setup.py

```diff
@@ -80,7 +80,7 @@ def build_native(spec):
         'doc' : ['sphinx', 'myst-parser[sphinx]', 'alabaster',
                  "sphinxcontrib-napoleon", "nbsphinx",
                  "ipython"],
-        '10x': ['bam2fasta==1.0.4'],
+        '10x': ['bam2fasta==1.0.8'],
```

Comment:
does the pysam problem goes away if we stick with `1.0.4`? Since we are deprecating it, no big reason to upgrade the dep here...

Reply:
it shouldn't, but heck, I'll give it a try ;)

Reply:
there you go - probably cleaner this way (fewer changes), but same failures!